corehq/apps/accounting/management/commands/hide_invoices_by_id.py

```diff
@@ -0,0 +1,38 @@
+# Use modern Python
+from __future__ import unicode_literals, absolute_import, print_function
+
+from optparse import make_option
+from django.core.management import BaseCommand
+
+from corehq.apps.accounting.models import Invoice
+
+
+class Command(BaseCommand):
+    help = 'Hides the specified invoice(s) from showing on reports'
+
+    option_list = BaseCommand.option_list + (
+        make_option('-u', '--unhide',
+                    action='store_true',
+                    default=False,
+                    dest='unhide',
+                    help="Make invoice(s) visible to the operations team"
```

Comment:
This is actually supposed to be visible. The option basically reverses what the command normally does.